### PR TITLE
Update HomeAssistant.gitignore

### PR DESCRIPTION
**Pull request by golles**
# Pull Request

Thank you for contributing to @toptal/gitignore and https://www.gitignore.io.

## New or update

Select the appropriate check box for this pull request. This helps when merging to ensure there are no conflicts with other templates or misunderstandings of how thee template list works.

### New

- [ ] Template - New `.gitignore` template
- [ ] Composition - Template made from smaller templates
- [ ] Inheritance - Template similar to an existing template
- [ ] Patch - Template extending functionality of existing template

### Update

- [x] Template - Update existing `.gitignore` template

## Details

I've added inline comments, explaining my changes

### CODE REVIEW

File: templates/HomeAssistant.gitignore [hunk 1]
```diff
@@ -2,7 +2,10 @@
 *.crt
 *.csr
 *.key
+.cloud
 .google.token
+.HA_VERSION
+.storage
```
Comment (golles) [inline templates/HomeAssistant.gitignore:8]: .storage contains all the user configuration files, including sensitive data. This shouldn't be checked in

File: templates/HomeAssistant.gitignore [hunk 2]
```diff
@@ -2,7 +2,10 @@
 *.crt
 *.csr
 *.key
+.cloud
```
Comment (golles) [inline templates/HomeAssistant.gitignore:5]: For me `.cloud` is empty, afaik this contains information about the user subscription with the online Nabu Casa service

File: templates/HomeAssistant.gitignore [hunk 3]
```diff
@@ -2,7 +2,10 @@
...
 .google.token
+.HA_VERSION
```
Comment (golles) [inline templates/HomeAssistant.gitignore:7]: `.HA_VERSION` contains the installed version of HA, in my opinion shouldn't be checked in

File: templates/HomeAssistant.gitignore [hunk 4]
```diff
@@ -11,13 +14,15 @@ home-assistant.db
 home-assistant_v2.db
 home-assistant_v2.db-*
 html5_push_registrations.conf
+image
```
Comment (golles) [inline templates/HomeAssistant.gitignore:17]: The `image` folder contains the profile images of all the person/accounts

File: templates/HomeAssistant.gitignore [hunk 5]
```diff
@@ -27,11 +32,11 @@ tts
 # automatically downloaded dependencies
 deps
 lib
-www
 
 # Log files
 home-assistant.log
+home-assistant.log.*
```
Comment (golles) [inline templates/HomeAssistant.gitignore:38]: For a while, HA creates 2 more log files (`home-assistant.log.1` and `home-assistant.log.fault`), these should also not be checked in

**Comment by golles** [comment]
Hi @alexmcode, is there anything else needed to get this change merged?

**Comment by konstrybakov** [comment]
> Hi @alexmcode, is there anything else needed to get this change merged?

Hey, @golles, Thank you for the contribution! I'll have a quick look and merge it asap

**Review by konstrybakov — APPROVED** [review]

**Comment by konstrybakov** [comment]
I've merged it, looks great, it'll take a bit to get to production, but expect soon!

**Pull request merged**